oojs.js

```diff
@@ -10,3 +10,31 @@
   // Have an artist property that is determined by some input.
 
 // Create an instance of the Album class.
+
+
+class Playlist {
+  constructor(title, songs) {
+    this.title = title;
+    this.songs = [];
+  }
+
+  this.favorites = 0;
```

Comment:
Everything in a class needs to live in a method, whether that's `constructor` or something else. In this example, you need to move `this.favorites = 0` into the constructor. Otherwise, it will error out.